server/src/collection.rs

```diff
@@ -95,4 +95,16 @@ impl Collection {
     pub fn path(&self) -> &OwnedObjectPath {
         &self.path
     }
+
+    pub async fn label(&self) -> String {
+        self.label.lock().await.clone()
+    }
+
+    pub async fn set_alias(&self, alias: String) {
+        *self.alias.lock().await = alias;
```

Comment:
Should emit a CollectionChanged as well

Reply:
Doesn't changing the alias means renaming the keyring file as well?

Reply:
> Should emit a CollectionChanged as well

No, it doesn't emit a CollectionChanged. At least gnome-keyring does not :)

Reply:
> Doesn't changing the alias means renaming the keyring file as well?

No, I don't think so. So, currently gnome-keyring names their keyring files based on the "label" property. And it doesn't rename them once we change the alias.